.github/workflows/docker-release.yml

```
@@ -0,0 +1,58 @@
+name: docker-release
+
+on:
+  push:
+    tags:
+      # Push events to matching v#.#.#*, ex: v1.2.3, v.2.4.6-beta
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=xmidt/${PWD##*/}
+          VERSION=edge
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/}
+          elif [[ $GITHUB_REF == refs/heads/* ]]; then
+            VERSION=$(echo ${GITHUB_REF#refs/heads/} | sed -r 's#/+#-#g')
+          elif [[ $GITHUB_REF == refs/pull/* ]]; then
+            VERSION=pr-${{ github.event.number }}
+          fi
+          TAGS="${DOCKER_IMAGE}:${VERSION}"
+          if [ "${{ github.event_name }}" = "push" ]; then
+            TAGS="$TAGS,${DOCKER_IMAGE}:sha-${GITHUB_SHA::8}"
+          fi
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1` | tail -1)
+          if [ "${LATEST_TAG}" == "${VERSION}" ]; then
+            TAGS="$TAGS,${DOCKER_IMAGE}:latest"
+          fi
+          echo ::set-output name=version::${VERSION}
+          echo ::set-output name=tags::${TAGS}
+          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```

```suggestion
          echo ::set-output name=created::$(date -u '+%c')
```
To keep it consistent with our makefile buildtime

---

This time isn't used for building 👍